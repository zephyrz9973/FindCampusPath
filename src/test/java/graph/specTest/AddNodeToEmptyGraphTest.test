#This is the test for adding node to the graph

#Create a graph
CreateGraph graph1

#Add one nodes
AddNode graph1 n1

#list the nodes the graph have
ListNodes graph1

#list the children of n1
ListChildren graph1 n1