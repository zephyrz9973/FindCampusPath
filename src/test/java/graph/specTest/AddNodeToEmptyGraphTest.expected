#This is the test for adding node to the graph

#Create a graph
created graph graph1

#Add one nodes
added node n1 to graph1

#list the nodes the graph have
graph1 contains: n1

#list the children of n1
the children of n1 in graph1 are: